pkg/storkctl/action.go

```diff
@@ -59,15 +65,54 @@ func newFailoverCommand(cmdFactory Factory, ioStreams genericclioptions.IOStream
 				}
 				_, err = storkops.Instance().CreateAction(&action)
 				if err != nil {
-					util.CheckErr(err)
-					return
+					printMsg(
+						fmt.Sprintf(
+							"failed to start failover for namespace %v due to error %v",
+							namespace, err),
+						ioStreams.ErrOut)
+					continue
+				}
+				printMsg(fmt.Sprintf("started failover for namespace %v", namespace), ioStreams.Out)
+				actions = append(actions, action)
+			}
+			for _, action := range actions {
+				isSuccessful, err := waitForActionToComplete(action.Name, action.Namespace)
+				if err != nil {
+					var errorMessage string
+					if _, ok := err.(*task.ErrTimedOut); ok {
+						errorMessage = fmt.Sprintf(
+							"timed out waiting for action %v/%v to complete\naction might still be scheduled/in-progress\n",
```

Comment:
nit: Since we are going to print this error on stdout, I feel we should start it with uppercase - `Timed out waiting.. `,  similar to how k8s does it
```
$ kubectl create -f pvc.yml
Error from server (AlreadyExists): error when creating "pvc.yml": persistentvolumeclaims "pvc" already exists
```